experimental/checkpoint.go

```diff
@@ -18,8 +24,25 @@ type Snapshotter interface {
 // EnableSnapshotterKey is a context key to indicate that snapshotting should be enabled.
 // The context.Context passed to a exported function invocation should have this key set
 // to a non-nil value, and host functions will be able to retrieve it using SnapshotterKey.
-type EnableSnapshotterKey struct{}
+//
+// Deprecated: use [WithSnapshotter] to enable snapshots.
+type EnableSnapshotterKey = ctxkey.EnableSnapshotterKey
```

Comment:
We are not using this `[symbol]` anywhere, so could you remove?

```suggestion
// Deprecated: use WithSnapshotter to enable snapshots.
type EnableSnapshotterKey = ctxkey.EnableSnapshotterKey
```

Reply:
Yeah, we're no longer using it, users might be.
My idea was to keep it deprecated for one (?) version, then remove it.

But we can remove immediately, since it's experimental. Just to confirm, is that the intention?

Reply:
no what I meant is literal characters `[` and `]`. see the suggested change

Reply:
Oh, yeah, those are [doc links](https://tip.golang.org/doc/comment#doclinks) which I'm slowing converting my code bases to. Reverting.